tutorials/02-conda_envs.rst

```diff
@@ -0,0 +1,114 @@
+An Astropy User's Guide to Managing Conda Environments and Jupyter Notebook
+===========================================================================
+
+
+    Help! My code library works in Python 2 and I don't have time to
+    make it work in Python 3
+
+Do not fear! This tutorial will teach you how to use conda environments
+to create a *separate* installation of Python 2.7. With conda
+environments, you can switch between Python 2 and 3 without having to
+worry about version conflicts.
+
+Step 1: Set up a Python 2 environment
+-------------------------------------
+
+-  Create a new ``conda`` environment
+
+   ::
+
+       conda create -n python2 python=2.7 anaconda 
+
+**NOTE:** ``anaconda`` at the end is optional...
```

Comment:
Maybe add some like "omitting it will make the environment smaller, but you will need to install the specific packages you need in the environment after activating it."

Reply:
Doing this.